README.md

```diff
@@ -51,6 +51,8 @@ const ComponentWithLocalStorage = () => {
 }
 ```
 
+**NOTE:** you can use `useLocalStorageNoSync` hook to opt out of using updates. First entry of the output array will then contain a function returning value stored in `localStorage` on demand.
```

Comment:
Let's turn this note into a fully featured section the following way:

- Let's turn _Tweaking update frequency_ headling level 3 and move it under _Basic usage_.

- Let's rename this section to _Tweaking **the** update frequency_.

- Split _Caveats_ into a _Caveats_ (level 3) subsection under _Basic usage_ describing the possible performance issue if the update frequency is low, and another level 2 section called _Limitations_ where the _Caveats_ section currently is, and keep the note about the type limitations there.

- Make a new section called _Disable syncing_ for the new hook, elaborate a little bit on its usage, and show a working example similar to the one for the other hook.

Reply:
Included in e8e98e2065fc0310fbe1fb02ac454c8199f5f7f2